### PR TITLE
Update FDA importer

### PR DESCRIPTION
**Pull request by kcotto**

### CODE REVIEW

**Comment by kcotto** [comment]
Addresses #427

File: lib/genome/importers/fda/fda.rb [hunk 1]
```diff
@@ -63,14 +63,14 @@ def create_drug_claims(row, gene_claim, fusion_protein)
 
   def create_interaction_claims
     CSV.foreach(file_path, encoding:'iso-8859-1:utf-8', :headers => true, :col_sep => "\t") do |row|
-      if row['Gene'].include?(':')
+      if row['Biomarker'].include?('-')
```
[the remark this answers is not in the document]
Comment (susannasiebert) [reply lib/genome/importers/fda/fda.rb]: Maybe they then need to be imported as two separate interaction claims?

**Comment by susannasiebert** [comment]
Ok, so outstanding decision to make:

- [ ] Decide whether or not to remove rows with Nonspecific (Congenital Methemoglobinemia) biomarker
- [ ] Decide whether or not to import rows with comma-separated biomarkers as separate interaction claims
- [ ] Decide whether or not to import the Therapeutic Area field as an interaction claim attribute.

I think refactoring this importer to work on the unmodified TSV is beyond the scope of this PR but I will make a separate issue for it because I think it would be worthwhile to make that change.

**Comment by ahwagner** [comment]
Thanks @kcotto and @susannasiebert.

Yes, remove any rows with biomarkers that are not single gene targets, including Nonspecific (etc).
Rows containing multiple comma-separated genes should be treated as separate interaction claims.
No, exclude the Therapeutic Area field as an attribute.

**Review by susannasiebert — APPROVED** [review]

**Pull request merged**